list.go

```diff
@@ -10,12 +10,25 @@ import (
 	"text/tabwriter"
 	"time"
 
+	"encoding/json"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/codegangsta/cli"
 	"github.com/opencontainers/runc/libcontainer"
 )
 
-const formatOptions = `table, json, yaml, xml, or "go=<go-template text>"`
+const formatOptions = `table or json`
+
+type containerList struct {
+	ContainerStates []*containerState `json:"containerlist"`
```

Comment:
don't add `list` to the name of things.  it can just be `containers` because we know its a list because of the `[]`